cumulus/client/block-builder/src/lib.rs

```diff
@@ -239,7 +239,7 @@ where
 			header.extrinsics_root().clone(),
 			HashFor::<Block>::ordered_trie_root(
 				self.extrinsics.iter().map(Encode::encode).collect(),
-				sp_core::storage::StateVersion::V1
+				sp_core::storage::StateVersion::V0 // TODO: switch to V1 once the upstream substrate switches.
```

Comment:
Why though? We use V1 everywhere else.

Reply:
Because the substrate we're using is still using V0 for t the extrinsics root https://github.com/paritytech/substrate/blob/04c7c6abef/frame/system/src/lib.rs#L142 . I uncovered this when I was testing around some stuff locally.